### PR TITLE
DOMA-1554 fix usage of attrs in multiPayment

### PR DESCRIPTION
**Pull request by toplenboren**
Refactor tests to actually test something

### CODE REVIEW

File: apps/condo/domains/acquiring/schema/MultiPayment.test.js [hunk 1]
```diff
@@ -143,11 +146,12 @@ describe('MultiPayment', () => {
 
                     const integrationClient = await makeClientWithNewRegisteredAndLoggedInUser()
                     await createTestAcquiringIntegrationAccessRight(admin, acquiringIntegration, integrationClient.user)
-                    const [_, updatedMultiPaymentAttrs] = await updateTestMultiPayment(integrationClient, multiPayment.id, {
+                    let [updatedMultiPayment] = await updateTestMultiPayment(integrationClient, multiPayment.id, {
```
Comment (SavelevMatthew) [inline apps/condo/domains/acquiring/schema/MultiPayment.test.js:149]: let?

And also why are you using raw?

**Review by SavelevMatthew — APPROVED** [review]

**Review by Dimitreee — APPROVED** [review]

File: apps/condo/domains/acquiring/access/MultiPayment.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ async function canManageMultiPayments ({ authentication: { item: user }, operati
 async function canReadMultiPaymentsSensitiveData ({ authentication: { item: user }, existingItem }) {
     if (user.deletedAt) return false
     if (user.isAdmin || user.isSupport) return true
-    if (await checkAcquiringIntegrationAccessRight(user, existingItem.integration)) return true
+    if (await checkAcquiringIntegrationAccessRight(user.id, existingItem.integration)) return true
```
Comment (dkoviazin) [inline apps/condo/domains/acquiring/access/MultiPayment.js:46]: Maybe 
```
const accessCheck = await checkAcquiringIntegrationAccessRight(user.id, existingItem.integration)
if (accessCheck) {
    return true
}
```

for debugging later :-)

**Review by dkoviazin — APPROVED** [review]

File: apps/condo/domains/acquiring/schema/MultiPayment.test.js [hunk 3]
```diff
@@ -143,11 +144,11 @@ describe('MultiPayment', () => {
...
```
Comment (akarjakin) [inline apps/condo/domains/acquiring/schema/MultiPayment.test.js:147]: please use `let`only if you need reassign value for a variable, in all other cases please use `const`

**Comment by toplenboren** [comment]
Was done in @SavelevMatthew PR, closing...

**Pull request closed without merging**